Makefile

```diff
@@ -115,12 +115,34 @@ verify: ## Verify code. Includes codegen, docgen, dependencies, linting, formatt
 		fi;}
 	actionlint -oneline
 
+openshift-verify: ## Verify code on OpenShift Prow CI. A stripped down copy of the "verify" target.
+	go mod tidy
```

Comment:
i usually run `go mod tidy && go mod vendor && go mod verify` when doing something like this. is there a reason to skip the vendor and verify steps?